helm/blueapi/templates/statefulset.yaml

```
@@ -44,15 +44,21 @@ spec:
           sources:
           - configMap:
               name: {{ include "blueapi.fullname" . }}-initconfig
+      - name: venv
+        emptyDir:
+          sizeLimit: 5Gi
+      {{- if .Values.initContainer.persistentVolume.enabled }}
+      - name: scratch
+        persistentVolumeClaim:
+          claimName: {{default "scratch-{{ .Values.image.tag | default .Chart.AppVersion }}" .Values.initContainer.persistentVolume.existingClaimName }}
+      {{- else }}
       - name: scratch-host
         hostPath:
           path: {{ required "initContainer enabled but no worker.scratch.root!" (.Values.worker.scratch).root }}
           type: Directory
-      - name: venv
-        emptyDir:
-          sizeLimit: 5Gi
       {{- end }}
-      {{- if .Values.debug.enabled }}
+      {{- end }}
+      {{- if or .Values.debug.enabled (and .Values.initContainer.enabled .Values.initContainer.persistentVolume.enabled)}}
```

Is the or at the correct place ?

```suggestion
      {{- if  .Values.debug.enabled or ( .Values.initContainer.enabled and .Values.initContainer.persistentVolume.enabled)}}
```

If both are identical in syntax I think the changes that I have suggested might be better just for readability

---

You could do `a | or b` but otherwise it has to be `or a b`: with the function first